### PR TITLE
Fix macOS update-test name

### PR DESCRIPTION
**Pull request by dduugg**
☝️

### CODE REVIEW

**Review by carlocab — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by dduugg** [comment]
hmm, i think someone with elevated access needs to do an update dance of the required checks…

**Comment by carlocab** [comment]
Ping @Homebrew/tsc

**Comment by Bo98** [comment]
Given we only test one macOS here, maybe just shorten it to `(macOS)` so we don't need to do this dance every year?

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -221,7 +221,7 @@ jobs:
         include:
           - name: update-test (Ubuntu 22.04)
             runs-on: ubuntu-22.04
-          - name: update-test (macOS 13)
+          - name: update-test (macOS 14)
```
Comment (dduugg) [inline .github/workflows/tests.yml:224]:
```suggestion
          - name: update-test (macOS)
```

**Pull request merged**